### PR TITLE
Link to the Migrating APIs page from the home page

### PR DESCRIPTION
**Pull request by nacin**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -52,11 +52,11 @@ HTTPS only guarantees the integrity of the connection between two systems, not t
 
 **Mixed Content:** Websites served over HTTPS need to ensure that all external resources (images, scripts, fonts, iframes, etc.) are also loaded over a secure connection. Modern browsers will refuse to load many insecure resources referenced from within a secure website. When migrating existing websites, this can involve a combination of automated and manual effort to update, replace, or remove references to insecure resources. For some websites, this can be the most time consuming aspect of the migration process.
 
-**APIs and Services:** Web services that serve primarily non-browser clients, such as web APIs, may require a more gradual and hands-on migration strategy, as not all clients can be expected to be configured for HTTPS connections or to successfully follow redirects.
+**APIs and Services:** Web services that serve primarily non-browser clients, such as web APIs, may require a [more gradual and hands-on migration strategy](https://https.cio.gov/apis/), as not all clients can be expected to be configured for HTTPS connections or to successfully follow redirects.
```
Comment (alex) [inline pages/index.md]: Is there a reason not to use a relative URL here?

**Comment by alex** [comment]
It looks like we're using the full URLs everywhere else, so :+1:

**Comment by konklone** [comment]
Er, I'd much prefer relative URLs, so that the links travel within the same repo in development, etc. I'll go fix any absolute URL links within the repo.

**Comment by alex** [comment]
@konklone Agreed on prefering relative, but I didn't want to penalize @nacin for a foolish consistency :D

**Comment by konklone** [comment]
Yeah, no I was actually the one who added the absolute URLs in the first place. ;) They were from before we had actual landing pages there -- I had some internal reason that made no sense.

In any case, fixed in 920f37d63e8bb30165f7b9bd3549a09be266d309.

**Comment by nacin** [comment]
I was only following the existing pattern. They should definitely be relative, so thanks for updating, @konklone.

**Pull request merged**